index.php

```diff
@@ -34,6 +34,7 @@
 $app->get(
     '/',
     function () {
+        $toto = $_SESSION['TOTO'] ?? 'NONE';
```

Comment:
Is this used by the tests?

Reply:
I added this to test session in the index.php. This file is not part of test but a minimal example of Slim's usage